librrd.h

```diff
@@ -33,6 +33,10 @@
 #define RRD_FILE_ERROR          3
 #define RRD_ERROR               4
 
+#define RRD_HEADER_SIZE         31
+#define DATASOURCE_VALUE_SIZE   8
+#define META_SIZE               4
+
 typedef enum { RRD_LOCAL_DOMAIN = 0, RRD_INTER_DOMAIN } rrd_domain;
```

Comment:
These are implementation details of the library that should not be exposed in its public interface.